libs/utils/utils.js

```diff
@@ -620,6 +620,24 @@ export function decorateAutoBlock(a) {
   });
 }
 
+const decorateCopyLink = (a, evt) => {
+  const userAgent = navigator.userAgent.toLowerCase();
+  const isMobile = /android|iphone|ipod|blackberry|iemobile|opera mini|mobile/.test(userAgent) && !/ipad/.test(userAgent);
+  if (!isMobile) {
+    a.remove();
+    console.log('Copy link is not supported for this device');
+    return;
+  }
+  const link = a.href.split(evt)[0];
```

Comment:
I was thinking whether we should simply `replace` the `evt` with `''`. Maybe the link is something like `https://www.adobe.com#_evt-copy?promoId=MAX`, case in which the `promoId` could be lost. It shouldn't be a regular occurrence, but it could lead to issues if an someone was not paying attention to the URL structure.

Reply:
Hi @overmyheadandbody 
I agree that `promoId` would be lost with the approach. In the above example, do we want the `promoId` to be a part of the URL when the user wants to copy and share it? The URL shared would be `https://www.adobe.com?promoId=MAX` in that case. If this is expected, then I can go ahead and replace the param.

Reply:
That should be confirmed with the feature's stakeholder. In my view we should keep query params. They might include some referrers, so I don't see why we would automatically remove them 🤷

Reply:
@overmyheadandbody As confirmed by the stakeholders, the query params should be reserved. Hence, I have made the required change. Thank you for your feedback.